pkgs/applications/editors/mindforger/default.nix

```diff
@@ -14,6 +14,8 @@ stdenv.mkDerivation rec {
 
   doCheck = true;
 
+  enableParallelBuilding = true ;
```

Comment:
Note: no harm on making it explicit.
But with cmake, meson and qmake this is actually the default. See https://nixos.org/nixpkgs/manual/#ssec-stdenv-attributes

Reply:
Oh, I missed that! Good to know for sure, thanks!